src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -54,13 +73,19 @@ class TelegramBotHandler extends AbstractProcessingHandler
     public function __construct(
         string $apiKey,
         string $channel,
+        string $parseMode = null,
+        bool $disableWebPagePreview = null,
+        bool $disableNotification = null,
```

Comment:
New args should be added to the end to be backwards compatible. And adding setters for those custom options is usually a good idea to allow for a more readable configuration code.. e.g. https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/SlackHandler.php#L192